src/pages/ConvertVrt/Form/index.tsx

```diff
@@ -0,0 +1,43 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Formik, FormikProps, FormikConfig } from 'formik';
+
+import getValidationSchema, { FormValues } from './validationSchema';
+
+export * from './validationSchema';
+
+export interface IAmountFormProps
+  extends Omit<FormikConfig<FormValues>, 'onSubmit' | 'initialValues'> {
+  onSubmit: (values: { vrt: string; xvs: string }) => Promise<void> | void;
+  children: (formProps: FormikProps<FormValues>) => React.ReactNode;
+  initialAmount?: string;
```

Comment:
I'd get rid of this prop and use `initialValues` directly since there's two inputs in this form